model/azure/registry.yaml

```diff
@@ -5,15 +5,15 @@ groups:
     brief: >
       This document defines generic attributes for Azure SDK.
     attributes:
-      - id: az.service_request_id
+      - id: azure.service.request.id
```

Comment:
Alternatives:
- `azure.server.request.id` (there is also `azure.client.request.id` counterpart not documented here) - this aligns with semconv better, but `service` gives a nuance that it's generated by the Azure service, while `server` is used in semconv  more the context of user app